### ISSUE TITLE
Is it possible to run postal from a console app?

### ISSUE THREAD

**Issue by curlyfro** [andrewdavey/postal#118]
i'm trying to use Postal from within NServiceBus console app but i'm getting an httpCapabilities cannot be null error.

does anyone know why?

**Comment by andrewdavey** [comment]
Have you followed the example here: http://aboutcode.net/postal/outside-aspnet.html ?

**Comment by mulgurul** [comment]
Yep, got the same problem, allthough it is only when you use dynamic Email.

If you use the stronly typed model, and call SendEmail(model) then it works.

Unfortunatly I really needed the dynamic method:-(

So great if it could be fixed!

BT Peter
